### PR TITLE
Python code added under Emulate Geo Location

### PR DESCRIPTION
**Pull request by Saketspradhan**
Working fine

**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Types of changes

- [ ] Change to the site (I am attaching a screenshot showing the before and after)
- [x] Code example added (for Python)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

**Review by baqua10 — APPROVED** [review]

**Review by baqua10 — DISMISSED** [review]

File: docs_source_files/content/support_packages/chrome_devtools.en.md [hunk 1]
```diff
@@ -29,7 +29,20 @@ public void geoLocationTest(){
 }  
   {{< / code-panel >}}
   {{< code-panel language="python" >}}
-# Please raise a PR to add code sample
+from selenium import webdriver
+from selenium.webdriver.chrome.service import Service
+
+PATH = "SPECIFY YOUR CHROMEDRIVER.EXE PATH HERE"
```
Comment (harsha509) [inline docs_source_files/content/support_packages/chrome_devtools.en.md]: Can you remove this line?

All code samples are written assuming that the driver paths are already set up in the machine.

File: docs_source_files/content/support_packages/chrome_devtools.en.md [hunk 2]
```diff
@@ -29,7 +29,20 @@ public void geoLocationTest(){
...
+
+def geoLocationTest():
+    driver = webdriver.Chrome(PATH)
```
Comment (harsha509) [inline docs_source_files/content/support_packages/chrome_devtools.en.md]: The above line can be modified if the path is removed!

**Review by harsha509 — CHANGES_REQUESTED** [review]
Hi @Saketspradhan,

Thanks for the PR.
I've requested for couple of changes, can you please look into it.

Also can you add the code sample to other translated pages (like es, zh-cn etc)

File: docs_source_files/content/support_packages/chrome_devtools.en.md (hunk 1 above)
Comment (Saketspradhan) [reply docs_source_files/content/support_packages/chrome_devtools.en.md]: Done.

**Comment by Saketspradhan** [comment]
Do I have to create separate PRs for every translated page?

**Comment by harsha509** [comment]
> Do I have to create separate PRs for every translated page?

No Please update the same PR with the code samples in other translated pages!

**Comment by harsha509** [comment]
Hi @Saketspradhan,

Did you get a chance to look into above comments?

**Review by harsha509 — APPROVED** [review]
Thank you @Saketspradhan!

I will add the code sample to other translated pages with another commit referring to this PR.

**Comment by harsha509** [comment]
Congratulations on your first PR to Selenium Docs @Saketspradhan 🎉 🎉 🥳 🥳

**Pull request merged**